docs/Parts List.md

```diff
@@ -0,0 +1,78 @@
+# Parts List
+
+## Printed Parts
+To Build Sawppy the Rover, you will need to print the following parts. Quantities and notes are listed.
+
+You will need about 3kg of filament to print all the parts.
+
+| QTY | PART | NOTES |
+|:---:|:----:|:-----:|
+| 88 | Clip2n125 | Print a few extra for spares |
+| 12 | Clip3n20 | Print a few extra for spares |
+| 6 | Wheel Hub | Print a few extra for spares |
+| 10 | LX-16A - Coupler | Print a few extra for spares |
+| 10 | LX-16A - Bracket | |
+| 6 | Wheel | |
+| 4 | Steering Knuckle | |
+| 4 | Body Corner | |
+| 2 | DiffBrace | |
+| 2 | DiffLink | |
+| 1 | DiffLower | |
+| 1 | DiffUpper | |
+| 2 | Rod Support | |
+| 1 | Power Panel | |
+| 2 | Fixed Knuckle | Mirrored |
+| 2 | Front Corner | Mirrored |
+| 2 | Rear Corner | Mirrored |
+| 2 | Bogie Wheels | Mirrored |
+| 2 | Bogie Body | Mirrored |
+| 2 | Rocker | Mirrored |
+| 2 | Rocker Body Mount | Mirrored |
+| 2 | DiffEnd | Mirrored |
+| 2 | Battery Tray | Mirrored |
+
+## Shafts
+The following are lengths and quantites for the shafts. You should make the differential axle from steel, and the others can be made from aluminum or steel.
+
+If you're careful and/or lucky, you can get all the shaft you need from a pair of 500mm shafts, not counting the 300mm shaft you need for the differential shaft.
+
+| QTY | LENGTH | PART |
+|:---:|:----:|:-----:|
+| 1 | 300 | Differential Shaft |
+| 6 | 50 | Wheel Axle Shaft |
+| 4 | 61 | Steering Shaft |
+| 2 | 66.5 | Suspension Bogie Pivot Point |
+| 2 | 84 | Suspension Rocker Pivot Point |
+
+## Hardware
+
+You will also need the following hardware
+
+| QTY | PART | NOTES |
+|:---:|:----:|:-----:|
+| 10 | LX-16A Servos | |
+| 30 | 608ZZ Bearings | |
+| 300 | M3x8mm socket head screws | |
+| 300 | M3 washers | |
+| 300 | M3 nuts | |
+| 100 | M3 threaded inserts | |
+| 100 | 5/16" external retaining clips (E-Clips) | |
+| 100 | M3x8 set screws | |
+| 2 | Turnbuckles | See above |
```

Comment:
What does "see above" refer to?

Reply:
That was a cross train of thought. I thought I was still on the main README page, where the info about turnbuckles is. Fixed with a reference back to that page.